### DISCUSSION TITLE
add node_indices to python impl

### DISCUSSION THREAD

**Discussion by Skino1337** [GothicKit/ZenKit#91] [Q&A]
Hi, python version dont have node_indices func (for MAN), can u update pypl? thx

**Comment by Skino1337** [comment]
zCModelAni_Header

    float samplePositionMin;
    float samplePositionScale;

    also not released(

**Comment by lmichaelis** [comment]
Hi there! Yes, it looks like I missed it when adding that API. I'll fix it as soon as I get the time, probably this weekend :) I'll let you know when it's updated.

**Reply by lmichaelis** [reply]
@Skino1337 I have release version [1.2.1](https://pypi.org/project/zenkit/1.2.1/) which adds `ModelAnimation.node_indices`.

**Reply by Skino1337** [reply]
Big thx.

**Comment by Skino1337** [comment]
Hi, I'll be waiting, thanks. And tell me, in order to unpack a position you need Vmin and Vmax, do you only have Vmin, or is it just named differently?

And tell me, do you already have ready-made functions for anpak?

Thank you!

![image](https://github.com/GothicKit/ZenKit/assets/5137675/4d057cef-db8b-4c57-badf-34d63c793912)

**Reply by lmichaelis** [reply]
You don't need to unpack the animation data as described [here](https://zk.gothickit.dev/engine/formats/animation/) when using ZenKit. It does it automatically. You can access position and rotation through `ModelAnimation.samples`

**Comment by Skino1337** [comment]
Hmmm, then you don’t know why there is such a big difference between the animation "curves".

![image](https://github.com/GothicKit/ZenKit/assets/5137675/b3a68262-e134-4ef0-b4d9-71353833d361)

**Reply by lmichaelis** [reply]
Well they're all on different axes so I'd expect them to be very different.

**Comment by Skino1337** [comment]
But this is "Bip01" x, y, z, on blender, and "Bip01" x, y, z, from man file, its the same anim, diff only frame count, asc recalc 24 to 60, but curve must be the same... I think u dont see blender curve on screen bottom.
